iam.tf

```diff
@@ -26,5 +26,13 @@ locals {
         "batcave/sonar-agent-api-key",
         "batcave/kiali",
         "batcave/gitlab-rails-secret-s3",
+        "batcave/alertmanager-secret",
+        "batcave/loki-write-keys",
+        "batcave/gitlab-secret",
+        "batcave/gitlab-rails-secret-backup"
```

Comment:
These gitlab keys will only be used in batcave clusters.  might want to make them conditionally added based on a far?

Reply:
Lets clean this up after release